src/App.tsx

```diff
@@ -1,7 +1,30 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string;
+};
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    key: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.pressedKey);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.pressedKey);
+  }
+
+  pressedKey = (event: KeyboardEvent) => {
+    this.setState({ key: `The last pressed key is [${event.key}]` });
+  };
+
+  render(): React.ReactNode {
+    return (
+      <div className="App">
+        <p className="App__message">{this.state.key}</p>
```

Comment:
```suggestion
  render(): React.ReactNode {
    const { pressedKet } = this.state;
    return (
      <div className="App">
        <p className="App__message">{pressedKey}</p>
```
You can make a destructuring to make your code a bit cleaner